### PR TITLE
Switch to Flagged Label & Terminology

### PR DESCRIPTION
**Pull request by Trenly**
cc @denelon
 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/109906&drop=dogfoodAlpha

### CODE REVIEW

**Comment by mdanish-kh** [comment]
@denelon How about a creating a new label `Package-Flagged`?

**Comment by stephengillie** [comment]
What should be used for the description and HTML color?

**Comment by Trenly** [comment]
> What should be used for the description and HTML color?

Same color as the Package-Hash-Flagged, description "The package must go through a more thorough validation to ensure it fully complies with repository policies"

File: .github/policies/flaggedPackages.yml [hunk 1]
```diff
@@ -69,13 +69,13 @@ configuration:
                   isRegex: False
         then:
           - addLabel:
-              label: Blocking-Issue
+              label: Validation-Hash-Flagged
```
Comment (stephengillie) [inline .github/policies/flaggedPackages.yml:72]:
```suggestion
              label: Package-Flagged
```

**Review by stephengillie — APPROVED** [review]

**Comment by mdanish-kh** [comment]
@stephengillie - Maybe there's a word limit to the label descriptions? I'm not seeing the full message on my end. Maybe we can make it more concise as: 
`Package needs further validation ensuring compliance with repository policies`

image

**Comment by stephengillie** [comment]
> @stephengillie - Maybe there's a word limit to the label descriptions? I'm not seeing the full message on my end. Maybe we can make it more concise as: `Package needs further validation ensuring compliance with repository policies`
> 
> image

Good catch.

**Pull request merged**